src/components/command-bar/commands/search/unified-search/components/unified-hits-container/index.tsx

```diff
@@ -14,58 +11,36 @@ import {
 } from '../../../components'
 // Unified search
 import { UnifiedHit } from '../unified-hit'
-import { gatherSearchTabsContent } from './helpers'
 // Types
-import type { ProductSlug } from 'types/products'
 import type { Hit } from 'instantsearch.js'
 import type { UnifiedSearchTabContent } from './helpers'
 // Styles
 import s from './unified-hits-container.module.css'
 
 /**
  * Render search results from our unified index into content-type tabs.
- *
- * Note: this component needs to be used within an `InstantSearch` container
- * imported from 'react-instantsearch-hooks-web'. That container provides
- * the context from which `rawHits` are pulled.
  */
 export function UnifiedHitsContainer({
```

Comment:
With `tabsData` logic now outside of this component, it's become much more purely presentational, taking the prepped `tabsData` and rendering the tabbed search results.